tests/testthat.R

```diff
@@ -5,8 +5,8 @@ library(quanteda)
 
 Sys.setenv("_R_CHECK_LENGTH_1_CONDITION_" = TRUE)
 
-ops <- quanteda_options()
 quanteda_options(reset = TRUE)
+#quanteda_options(threads = 2)
```

Comment:
I remember the problem, but it seems that other RcppParallel dependent packages are not doing anything to limit the number of threads. But we can set the number before running the tests if we really need.

Reply:
The problem is that the examples run before the tests. So we'd need to do this before anything at all started to run, including vignettes.

Reply:
I think examples are fine, because they don't require a lot of computing power, but vignettes might not. How about including `quanteda_options(threads = 2)` in the hidden code chunk in every Rmd file?

Reply:
Also posted a quetsion https://github.com/RcppCore/RcppParallel/issues/110